ignf_gpf_sdk/store/StoredData.py

```diff
@@ -18,3 +21,26 @@ class StoredData(TagInterface, CommentInterface, SharingInterface, EventInterfac
     STATUS_GENERATED = "GENERATED"
     STATUS_DELETED = "DELETED"
     STATUS_UNSTABLE = "UNSTABLE"
+
+    def delete_cascade(self, before_delete: Optional[Callable[[List["StoreEntity"]], List["StoreEntity"]]] = None) -> None:
+        """suppression en cascade des offres : uniquement les offres
+
+        Args:
+            before_delete (Optional[Callable[[List[StoreEntity]], List[StoreEntity]]], optional): fonction à lancé avant la suppression entrée liste des entités à supprimé,
+                sortie liste définitive des entités à supprimer. Defaults to None.
+        """
+        # suppression d'une stored_data : uniquement l'upload, configuration et stored_data
```

Comment:
Je ne comprends pas ce commentaire.

Reply:
```suggestion
        # suppression d'une stored_data : l'upload, configuration et stored_data
```
un "copier coller modifier" qui est mal passé ...

Reply:
Du coup c'est même offering, configuration et stored_data

Reply:
```suggestion
        # suppression d'une stored_data : uniquement l'offering, configuration et stored_data
```
en effet ...

Reply:
```suggestion
        # suppression d'une stored_data : offering et configuration liées puis la stored_data
```